frontend/src/components/projects/ProjectsListContent.js

```diff
@@ -121,6 +122,7 @@ const ProjectsListContent = ({
   return <>
     <AppPageHeader title="Projects" extra={[
       <AddButton key='add' url="/projects/add" />,
+      <LinkButton key="link" url="/projects/actions/0" />,
```

Comment:
Maybe use a constant for the URL since this is hard to identify what the action 0 is looking at this. There might also be other actions added down the line and it would help make sure we point to the right one.

Reply:
There is definitely some method to do that automatically somewhere. The back-end viewsets send a list of actions, and there is a front-end 'helper' that translates those actions into buttons/urls

Reply:
Lookup actionsToButtonList in the front-end, this is what is used in the front-end for template actions

Reply:
I think Sebastian wanted a custom button with a link icon...

Reply:
Aren't we in ProjectListContent with the same list of buttons as in the other sections?